lib/Mage/System/Ftp.php

```diff
@@ -308,7 +308,7 @@ public function close()
      *
      * @param $mode
      * @param $remoteFile
-     * @return bool
+     * @return int
```

Comment:
https://www.php.net/manual/en/function.ftp-chmod.php
> Returns the new file permissions on success or false on error. 
```suggestion
    * @return int|false
```